apps/yggdrasil/test/yggdrasil/character_test.exs

```diff
@@ -0,0 +1,156 @@
+defmodule Yggdrasil.CharacterTest do
```

Comment:
We need to figure out a way to organize our tests more. We'll end up with several functions in a module and need to have an easy way to group them. Is it possible to have sub-modules with tests?

```Elixir
defmodule Yggdrasil.CharacterTest do
    use ExUnit.Case, async: false

    defmodule Yggdrasil.CharacterTests.CreateChangesetTest do
        use ExUnit.Case, async: false

        test "it requires a valid user id" do
        end
    end
end
```

I'm just thinking that as we add functions this file will get ugly. Also I greatly prefer the BDD spec style test names. So I favor "it requires the character name to be at least 3 characters" as a test name over "invalid character name length < 3".